internal/docker/runner.go

```diff
@@ -143,8 +149,21 @@ func (r *Runner) setup(suite config.Suite, run config.Run) error {
 	return nil
 }
 
-// run runs the tests defined in the config.Project.
-func (r *Runner) run() (int, error) {
+func (r* Runner) beforeExec(tasks []string) (error) {
```

Comment:
Redundant parentheses.
```suggestion
func (r* Runner) beforeExec(tasks []string) error {
```